test/functional/testlib.bash

```diff
@@ -1229,6 +1229,50 @@ sign_manifest() { # swupd_function
     -outform DER -out "$(dirname "$manifest")"/Manifest.MoM.sig
 }
 
+# Signs the latest_versin with a PEM key and generates the signed latest_version in the same location
+# Parameters:
+# - latest_version: the path to the latest_version to be signed
+sign_latest_version() {
+	local latest_version=$1
+
+	# If no parameters are received show usage
+	if [ $# -eq 0 ]; then
+		cat <<-EOM
+			Usage:
+			    sign_latest_version <latest_version>
+			EOM
+		return
+	fi
+	validate_item "$latest_version"
+
+	sudo openssl smime -sign -binary -in "$latest_version" \
+		-signer "$FUNC_DIR"/Swupd_Root.pem \
+		-inkey "$FUNC_DIR"/private.pem \
+		-outform DER -out "$(dirname "$latest_version")"/latest_version.sig
+}
+
+# Signs the latest with a PEM key and generates the signed latest.sig in the same location
+# Parameters:
+# - latest: the path to the latest to be signed
+sign_format_version() {
```

Comment:
isn't this function exactly the same as the function above? you can use the same function to sign both files

Reply:
They do exactly the same thing but for different files. I guess I can parameterize this and reduce to just one function

Reply:
Done